code/game/objects/structures/petrified_statue.dm

```diff
@@ -67,11 +67,21 @@
 	return ..()
 
 /obj/structure/statue/petrified/deconstruct(disassembled = TRUE)
+	var/destruction_message = "[src] shatters!"
 	if(!disassembled)
 		if(petrified_mob)
 			petrified_mob.investigate_log("has been dusted by statue deconstruction.", INVESTIGATE_DEATHS)
+			if(iscarbon(petrified_mob))
+				var/mob/living/carbon/petrified_carbon = petrified_mob
+				var/obj/item/organ/internal/brain/carbon_brain = petrified_carbon.getorganslot(ORGAN_SLOT_BRAIN)
+				carbon_brain.Remove(petrified_carbon)
+				carbon_brain.forceMove(get_turf(src))
+				carbon_brain.name = "petrified [carbon_brain.name]"
+				carbon_brain.desc = "[carbon_brain.desc] This one seems a bit more... smooth than a normal brain. Probably'd still work."
+				carbon_brain.add_atom_colour(list(rgb(77,77,77), rgb(150,150,150), rgb(28,28,28), rgb(0,0,0)), FIXED_COLOUR_PRIORITY)
+				destruction_message = "[src] shatters, a solid brain tumbling out!"
```

Comment:
Please add a new stone brain, instead of hardcoding it.
Something like this can be used to brainswap
```
				var/obj/item/organ/internal/brain/stone/stone_brain = new()
				old_brain.before_organ_replacement(stone_brain) //move traumas and stuff
				old_brain.Remove(petrified_carbon, special=TRUE, no_id_transfer=TRUE) //mob is without brain for a sec
				stone_brain.Insert(petrified_carbon, special=TRUE, drop_if_replaced=FALSE) //insert new brain
				stone_brain.Remove(petrified_carbon) //drop the brain, this time the player follows the brain
				stone_brain.forceMove(get_turf(src)) //take from nullspace and put on floor
```

Reply:
bad idea

Reply:
Can you actually explain why? We’re trying to move away from hardcoded organs

Reply:
I talked with Fikou on discord and there’s a few things like psykers which would be incompatible then. This is fine for now but will need to be reworked eventually